addons/web/controllers/main.py

```diff
@@ -46,23 +46,12 @@
 
 _logger = logging.getLogger(__name__)
 
-if hasattr(sys, 'frozen'):
```

Comment:
As we keep jinja for some extra tools (pos), why removing this ? Unless decision was to remove it finally ?